cachito/workers/scm.py

```diff
@@ -72,10 +72,11 @@ def _create_archive(self, from_dir):
         with tarfile.open(self.sources_dir.archive_path, mode="w:gz") as bundle_archive:
             bundle_archive.add(from_dir, "app")
 
-    def clone_and_archive(self):
+    def clone_and_archive(self, gitsubmodule):
```

Comment:
The other functions have `gitsubmodule` as a keyword arg, here it is a positional arg. I'd like to keep it consistent (my preference is keyword arg)